vnext/Desktop/JSI/ChakraJsiRuntime_core.cpp

```diff
@@ -1,11 +1,13 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-#include "ChakraRuntime.h"
-#include "ChakraRuntimeFactory.h"
+//#include "ChakraRuntime.h"
```

Comment:
remove commented out code (here and below)

Reply:
As a reminder, this is still a draft.

I will address all feedback before publishing it as a "stable" pull request.